### PR TITLE
Update actions e2e test to cover print_event with multiple events

### PR DESCRIPTION
**Pull request by ttuffin**
Closes [AAP-5634](https://issues.redhat.com/browse/AAP-5634).

### CODE REVIEW

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by Alex-Izquierdo — COMMENTED** [review]
3.9 test fails, not sure why

**Comment by ttuffin** [comment]
> 3.9 test fails, not sure why

The process timed out. Not sure why, trying to reproduce locally but so far unable to

**Comment by Alex-Izquierdo** [comment]
can you add `-vv` here? https://github.com/ansible/ansible-rulebook/blob/main/.github/actions/e2e-tests/action.yml#L9

**Comment by Alex-Izquierdo** [comment]
It's a time race. Let's decrease the event delay

File: tests/e2e/config/default.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-event_delay: 1
+default_event_delay: 0.75
```
Comment (Alex-Izquierdo) [inline tests/e2e/config/default.yml:1]: Is this value not too tight? I would feel better with a more relaxed delay :)

File: tests/e2e/test_variables.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 LOGGER = logging.getLogger(__name__)
 DEFAULT_CMD_TIMEOUT = SETTINGS["cmd_timeout"]
-DEFAULT_EVENT_DELAY = SETTINGS["event_delay"]
+DEFAULT_EVENT_DELAY = SETTINGS["default_shutdown_after"]
```
Comment (Alex-Izquierdo) [inline tests/e2e/test_variables.py:15]: Wrong value

**Review by Alex-Izquierdo — CHANGES_REQUESTED** [review]

File: tests/e2e/test_variables.py (hunk 2 above)
Comment (ttuffin) [reply tests/e2e/test_variables.py:15]: omg /o\

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by mkanoor — APPROVED** [review]

**Pull request merged**